apps/drec-api/migrations/1633366638472-DeviceGroupAttributes.ts

```diff
@@ -0,0 +1,105 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DeviceGroupAttributes1633366638472 implements MigrationInterface {
+  name = 'DeviceGroupAttributes1633366638472';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "device_group" ADD "fuelCode" character varying NOT NULL`,
```

Comment:
just a note, this will fail on existing environment

Reply:
hmm, why? I generated the migration and ran it on top of the current environment and it worked. The device group table is also empty.

Reply:
if the table is empty then it's ok